linkerd/proxy/transport/src/tls/client.rs

```diff
@@ -101,36 +101,25 @@ where
             *self = match self {
                 ConnectFuture::Init { future, tls } => {
                     let io = try_ready!(future.poll());
-                    let remote_addr = io.remote_addr().expect("socket must have remote addr");
 
                     match tls {
-                        Conditional::Some((server_name, local_tls)) => {
-                            trace!("initiating TLS to {}", server_name.as_ref());
-                            let future =
+                        Conditional::Some((peer_identity, local_tls)) => {
+                            trace!(sni = %peer_identity.as_ref(), "initiating TLS");
```

Comment:
nit/tioli: should this be 
```suggestion
                            trace!(peer.sni = %peer_identity.as_ref(), "initiating TLS");
```

Reply:
actually, this is all redundant with value already in the context...